drivers/video/fbdev/r61520fb.c

```diff
@@ -396,10 +402,81 @@ static irqreturn_t lcdc_irq_handler(int irq, void *arg)
   return IRQ_HANDLED;
 }
 
+
+static void readReg(uint32_t reg, uint8_t n, const char *msg) //this is for debugging registers on TFT panels
+{
+  uint32_t res;
+  lcdc_wr_cmd(reg);
+  printk("REG 0x%02x:     ", reg);
+  for(x=0; x<n; x++){
+    res = lcdc_rd_dat();
+    printk(KERN_CONT "%02x ",res);
+  }
+  printk(KERN_CONT "%s ",msg);
+}
+
+
+
+
+
 static int panel_init(void)
 {
   uint16_t x, ver[4]={0};
   
+  suniv_setbits(iomm.lcdc + PE_DATA, (11 << 0));
+  mdelay(50);
+  suniv_clrbits(iomm.lcdc + PE_DATA, (11 << 0));
+  mdelay(150);
+  suniv_setbits(iomm.lcdc + PE_DATA, (11 << 0));
+  mdelay(150);
+  
+  printk(" ");
+  printk("TFT IDENTIFICATION 8BITS:");
+  printk(" ");
```

Comment:
printk(" ");
  printk("TFT IDENTIFICATION 8BITS:");
  printk(" ");

should be also under debug